### PR TITLE
10 get reviews queries

### PR DESCRIPTION
**Pull request by 16000psi**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -94,14 +94,96 @@ describe("GET /api/reviews", () => {
...
+    .then(({body}) => {
+        const {reviews} = body
+        reviews.forEach((review) => {
+            expect(review.category).toBe("social deduction")
+        })
+        
+    })
+    })
+    test("Incorrect category returns 200 with no results", () => {
+        return request(app)
```
Comment (pippaamy) [inline __tests__/app.test.js:120]: this needs to be a 400 :)

**Review by pippaamy — COMMENTED** [review]
Couple of changes to make in your tests1 :)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -94,14 +94,95 @@ describe("GET /api/reviews", () => {
...
+            .then(({body}) => {
+                const {reviews} = body
+                expect(reviews).toBeSortedBy("designer", {descending:true})
+            
+            })
+        })
+    })
+    test("Invalid sort by query returns 400 error", () => {
+        return request(app)
+        .get("/api/reviews?sort_by=evilSqlHack")
```
Comment (Augs0) [inline __tests__/app.test.js:151]: 😈 👀

File: __tests__/app.test.js (hunk 1 above)
Comment (Augs0) [reply __tests__/app.test.js:120]: This should be 404 as any string has the potential to be a category BUT we might not have it

File: __tests__/app.test.js [hunk 3]
```diff
@@ -94,14 +94,95 @@ describe("GET /api/reviews", () => {
...
+            return request(app)
+            .get("/api/reviews?sort_by=designer")
+            .expect(200)
+            .then(({body}) => {
+                const {reviews} = body
+                expect(reviews).toBeSortedBy("designer", {descending:true})
+            
+            })
+        })
+    })
```
Comment (Augs0) [inline __tests__/app.test.js:148]: We probably need a test for a category that exists but has no reviews yet. This should return an empty array and 200

**Review by Augs0 — CHANGES_REQUESTED** [review]
Great work here Dave, just a. few small changes

File: __tests__/app.test.js (hunk 3 above)
Comment (16000psi) [reply __tests__/app.test.js:148]: Thanks for the feedback August! I've incorporated that stuff now :)

**Pull request merged**